docs/registry/rhoas-cli-service-registry/README.adoc

```diff
@@ -380,45 +280,38 @@ After you register a {registry} artifact, you can download the artifact to updat
 
 .Prerequisites
 
-* You have created a {registry} instance with at least one artifact.
+* You've created a {registry} instance with at least one artifact.
 
 .Procedure
 
-Download the artifact in either of the following ways:
-
-* Use the `rhoas service-registry artifact get` command and specify the artifact ID.
-* Use the `rhoas service-registry artifact download` command and specify the global ID.
-
+* Download the artifact by specifying either the artifact ID or the global ID.
```

Comment:
In the original text, I used a bullet list to highlight that you use a different command depending on which ID you use ("get" for artifact ID, "download" for global ID). I don't think that distinction is clear in this rewrite?

Reply:
I tried to bring that procedure more in line with the modular documentation standards, where a single step is represented by a bullet point. This particular procedure is a single step because there's only one action to take (download the artifact), but there're two options for completing it. The other way to handle this would be to use sub-bullets, which seems to be more complicated to follow:

* Download the artifact by doing one of the following:
    * Download the artifact by artifact ID...
    * Download the artifact by global ID...

Does that seem to flow better? If so, I'm happy to change it.

Reply:
Thanks, @bhardesty -- my apologies, I'd forgotten that a single step had to be a bullet point. In that case, I'd prefer to keep my original text, and just apply the bullet point and the sub-bullet points to that text? I prefer this approach to avoid the repetition of "Download the artifact by", and make it easier to see that different commands are used (and also to ensure that "following" is used as an adjective before a noun). Have I convinced you, or would you still prefer the one-line version? :)

Reply:
No problem - I made that update.